File: mods/login/src/main/java/com/boardour/login/LoginProviderImpl.java
package com.boardour.login;

import android.content.Context;

import com.alibaba.android.arouter.facade.annotation.Route;
import com.boardour.comm.RouterPath;
import com.boardour.comm.ft_login.ILoginProvider;

@Route(path = RouterPath.LOGIN_PROVIDER)
public class LoginProviderImpl implements ILoginProvider {
    @Override
    public void init(Context context) {

    }

    @Override
    public boolean isLogin() {
        return true;
    }

    @Override
    public String userName() {
        return "lven";
    }
}

File: mods/login/src/main/java/com/boardour/login/LoginProviderImpl.kt
package com.boardour.login

import android.content.Context
import com.alibaba.android.arouter.facade.annotation.Route
import com.boardour.comm.RouterPath
import com.boardour.comm.ft_login.ILoginProvider

/**
 * 登录模块实现类
 */
@Route(path = RouterPath.LOGIN_PROVIDER)
open class LoginProviderImpl : ILoginProvider {

    override fun init(context: Context?) {

    }

    override fun isLogin(): Boolean = true

    override fun userName(): String? {
        return "kotlin lven"
    }


}